### PR TITLE
Add basic sentry error reporting to bootstrap

### PR DESCRIPTION
**Pull request by leth**
Fixes https://github.com/weaveworks/launcher/issues/96

### CODE REVIEW

File: bootstrap/main.go [hunk 1]
```diff
@@ -74,7 +81,9 @@ func main() {
 }
 
 func die(msg string, args ...interface{}) {
-	fmt.Fprintf(os.Stderr, msg, args...)
+	formatted := fmt.Sprintf(msg, args...)
+	fmt.Fprintf(os.Stderr, formatted)
+	raven.CaptureMessage(formatted, map[string]string{})
```
Comment (aaron7) [inline bootstrap/main.go]: Could we pass the scheme and hostname options here to help debug dev vs prod?

File: bootstrap/main.go [hunk 2]
```diff
@@ -24,6 +26,11 @@ type options struct {
 	Token     string `long:"token" description:"Weave Cloud token" required:"true"`
 }
 
+func init() {
+	// https://sentry.io/weaveworks/launcher-bootstrap/
+	raven.SetDSN("https://44cf71b08710447888c993011b1302fc:8f57948cabd34bbe854b196635bff59f@sentry.io/288665")
+}
+
 func main() {
```
Comment (aaron7) [inline bootstrap/main.go]: Capturing panics would be useful:
https://github.com/weaveworks/launcher/blob/master/agent/main.go#L154

**Comment by leth** [comment]
> Perhaps we could have a error function which is used for non-user related errors which calls sentry and then calls die? It may help reduce any error fatigue.

Good point, I agree.

> Currently die is also being used for user actionable errors like Could not find kubectl in PATH, please install it.

Maybe it'd still be nice to know how often some errors (e.g. this one) happen?

**Comment by leth** [comment]
> Maybe it'd still be nice to know how often some errors (e.g. this one) happen?

Longer term, we should use segement to track this kind of metric!

**Pull request merged**